.travis.yml

```diff
@@ -4,7 +4,9 @@ sudo: required
 branches:
   only:
   - master
-  - develop	
+  - develop
+  - release-1.0
+  - pre-release-v1.0.8
```

Comment:
What is this `pre-release-v1.0.8` branch (on its face, this looks inconsistent with the git flow) and does the travis config file need to be updated to include it? I can accept that we would include the current release branch (though this might warrant a conversation with @pgev), but I'm not sure that we should update the travis for every other branch.

Reply:
Removed it.